api/vpc/v1alpha2/vpc_types.go

```diff
@@ -81,6 +81,14 @@ type VPCDHCP struct {
 	Range *VPCDHCPRange `json:"range,omitempty"`
 	// PXEURL (optional) to identify the pxe server to use to boot hosts connected to this segment such as http://10.10.10.99/bootfilename or tftp://10.10.10.99/bootfilename, http query strings are not supported
 	PXEURL string `json:"pxeURL,omitempty"`
+	// DNS server address to configure Domain Name server for this particular segment such as 10.10.10.2
+	DNSServer string `json:"dnsServer"`
+	// TimeServer address to configure NTP time server for this particular segment such 10.10.10.2
+	TimeServer string `json:"timeServer"`
+	// InterfaceMTU to configure the MTU for the device which send a DHCP discovery.
+	// +kubebuilder:validation:Minimum: 96
+	// +kubebuilder:validation:Maximum: 9036
+	InterfaceMTU uint32 `json:"interfaceMTU"`
```

Comment:
Is it optional as well? Additionally, please extend the description to clearly state that it will not affect the MTU configured on the switch ports and will be just added to the DHCP packets and may or may not be handled by the client.